libdebug/architectures/aarch64/aarch64_ptrace_register_holder.py

```diff
@@ -156,6 +156,14 @@ def provide_regs_class(self: Aarch64PtraceRegisterHolder) -> type:
         """Provide a class to hold the register accessors."""
         return Aarch64Registers
 
+    def provide_regs(self: Aarch64PtraceRegisterHolder) -> list[str]:
```

Comment:
These two new functions should be added in the definition of the abstract PtraceRegisterHolder